frontend/.nvmrc

```diff
@@ -1 +1 @@
-10.15.3
+14.21.3
```

Comment:
Do you plan on upgrading to the LTS (18.14.2) in a future PR?

Node 14 will hit end of life in a month: https://nodejs.dev/en/about/releases/

Reply:
Yes, but It will be part of the Next.js migration where it will be easier